main.cpp

```diff
@@ -115,21 +120,38 @@ int main()
     /*
      construct 2 more Car instances via emplace_back.
      */
-    
+    cars.emplace_back("phil");
+    cars.emplace_back("angela");
     /*
      now reserve and emplace_back your Trucks and Motorcyles
      */
+    motorcycles.reserve(2);
+    motorcycles.emplace_back("jerry");
+    motorcycles.emplace_back("george");
+
+    trucks.reserve(4);
+    trucks.emplace_back("kramer");
+    trucks.emplace_back("elaine");
+    trucks.emplace_back("newman");
+    trucks.emplace_back("estelle");
     
     
-    
-    
-    assert(false);
+    //assert(false);
     //add the cars, motorcycles and trucks to the highway using range-based for() loops: for( element : vec ) { ... }
     //be careful to not accidentally make element copies when iterating.
     
     HighwayPatrol cop;
     cop.scanHighway(&highway);
 
```

Comment:
The cop needs to scan the highway AFTER all vehicles are added.
Right now, they are scanning an empty highway.
- [ ] revise